spec.md

```diff
@@ -589,24 +598,24 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id/ser
+| 422 Unprocessable Entity  | MUST be returned if the broker requires that <code>app_guid</code> be included in the request body. The expected response body is: <code>{ "error": "RequiresApp", "description": "This service supports generation of credentials through binding an application only." }</code>  |
 
 Responses with any other status code will be interpreted as a failure and an unbind request will be sent to the broker to prevent an orphan being created on the broker. Brokers can include a user-facing message in the `description` field; for details see [Broker Errors](#broker-errors).
 
 ##### Body #####
 
-All response bodies must be a valid JSON Object (`{}`). This is for future compatibility; it will be easier to add fields in the future if JSON is expected rather than to support the cases when a JSON body may or may not be returned.
+All response bodies MUST be a valid JSON Object (`{}`). This is for future compatibility; it will be easier to add fields in the future if JSON is expected rather than to support the cases when a JSON body might or might not be returned.
 
 For success responses, the following fields are supported. Others will be ignored. For error responses, see [Broker Errors](#broker-errors).
 
 |  Response Field | Type  | Description  |
 |---|---|---|
-| credentials  | object  |  A free-form hash of credentials that may be used by applications or users to access the service. |
-| syslog\_drain_url  | string  | A URL to which logs may be streamed. <code>"requires":["syslog\_drain"]</code> must be declared in the [Catalog](#catalog-mgmt) endpoint or the platform should consider the response invalid.  |
-| route\_service_url  | string  |  A URL to which the platform may proxy requests for the address sent with <code>bind\_resource.route</code> in the request body. <code>"requires":["route\_forwarding"]</code> must be declared in the [Catalog](#catalog-mgmt) endpoint or the platform should consider the response invalid. |
-| volume\_mounts  | array-of-objects  | An array of configuration for mounting volumes. <code>"requires":["volume_mount"]</code> must be declared in the [Catalog](#catalog-mgmt) endpoint or the platform should consider the response invalid.  |
+| credentials  | object  |  A free-form hash of credentials that can be used by applications or users to access the service. |
+| syslog\_drain_url  | string  | A URL to which logs MUST be streamed. <code>"requires":["syslog\_drain"]</code> MUST be declared in the [Catalog](#catalog-mgmt) endpoint or the platform can consider the response invalid.  |
+| route\_service_url  | string  |  A URL to which the platform MUST proxy requests for the address sent with <code>bind\_resource.route</code> in the request body. <code>"requires":["route\_forwarding"]</code> MUST be declared in the [Catalog](#catalog-mgmt) endpoint or the platform can consider the response invalid. |
```

Comment:
I think you're correct, but swapping this for a MUST here feels awkward to me - and this isn't the only spot. I'm not sure what saying "MUST consider the response invalid" means. Does it mean they MUST reject the msg? Can they do some kind of error recovery?  I would prefer to talk about what MUST or MUST NOT be in the message and then add a section that talks about what the platform is to do when it gets a non-compliant message in general (assuming that covers all of the cases).  But we can do that in a follow-on PR.

But, in the meantime... I'll make this a MUST since that's the  biggest change I'd like to make this PR on this topic and I think its a step in the right direction.